modules/openapi-generator/src/main/resources/python/python-experimental/model_templates/method_init_composed.mustache

```diff
@@ -17,6 +17,11 @@
         self._path_to_item = _path_to_item
         self._configuration = _configuration
 
+        if self._configuration.discard_unknown_keys:
+            for property_name in list(kwargs.keys()):
+                if property_name not in self.attribute_map:
+                    del kwargs[property_name]
```

Comment:
Can you move this new code and the 
```
        self._data_store = {}
        ...
        self._configuration = _configuration
```
into method_init_shared?
It is always used together and defined only once if we put it there.

Reply:
Done.